orbit/pkg/update/runner.go

```diff
@@ -176,12 +209,6 @@ func (r *Runner) UpdateAction() (bool, error) {
 			}
 			log.Info().Str("target", target).Msg("update completed")
 			didUpdate = true
-			if target == "nudge" {
-				r.mu.Lock()
-				r.localHashes["nudge"] = metaHash
```

Comment:
There were some quirks that led me to this approach that I'm not sure are captured by the new `StoreLocalHash` approach. I will double-check locally.